### PR TITLE
LWC component for confirmation screen on OOBO flow

### PR DESCRIPTION
**Pull request by shubhamk67**
Added LWC for confirmation screen on OOBO flow to display Payment Link details.

### CODE REVIEW

File: force-app/main/default/lwc/adyenPblConfirmationScreen/adyenPblConfirmationScreen.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    
+                </div>
+            </div>
+        </template>
+
+        <template lwc:if={isCardPaymentMode}>
+            <div class="slds-text-align_center">
+                <lightning-icon class="slds-m-around_medium" icon-name="action:approval" alternative-text="Success" size="xx-small"></lightning-icon>
+                <h1 class="slds-text-heading_large">Success!</h1>
+                <div class="slds-align_absolute-center slds-m-around_medium">Your Order has been created and Card has been charged {currencyIsoCode} {amount}</div>
```
Comment (dcardos) [inline force-app/main/default/lwc/adyenPblConfirmationScreen/adyenPblConfirmationScreen.html:86]: I believe it should be order with lower case "O" and card with lower case "C".

File: force-app/main/default/lwc/adyenPblConfirmationScreen/adyenPblConfirmationScreen.js-meta.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>61.0</apiVersion>
+    <isExposed>true</isExposed>
+    <targets>
+        <target>lightning__FlowScreen</target>
+    </targets>
+    <targetConfigs>
+        <targetConfig targets="lightning__FlowScreen">
+            <property name="shopperName" type="String" label="shopperName" />
```
Comment (dcardos) [inline force-app/main/default/lwc/adyenPblConfirmationScreen/adyenPblConfirmationScreen.js-meta.xml:10]: I think you meant label as Shopper Name.

**Review by dcardos — CHANGES_REQUESTED** [review]
Great LWC, very clean and well coded! I still prefer using lightning layout than the grid classes so it is more readable, but it is more of a personal preference.
Regarding the changes, only very minor stuff.
PS: I think later we will need to move all the texts to labels so we allow merchants to do theirs translations.  Me might also need to stop the flow from proceeding when in an error state.

File: force-app/main/default/lwc/adyenPblConfirmationScreen/adyenPblConfirmationScreen.js-meta.xml [hunk 3]
```diff
@@ -7,7 +7,7 @@
     </targets>
     <targetConfigs>
         <targetConfig targets="lightning__FlowScreen">
-            <property name="shopperName" type="String" label="shopperName" />
+            <property name="shopperName" type="String" label="shopper Name" />
```
Comment (dcardos) [inline force-app/main/default/lwc/adyenPblConfirmationScreen/adyenPblConfirmationScreen.js-meta.xml:10]: I think for consistency it should be Shopper with capital S, almost there. haha

**Review by dcardos — CHANGES_REQUESTED** [review]

**Review by dcardos — APPROVED** [review]

**Review by zenit2001 — APPROVED** [review]

**Pull request merged**